### PR TITLE
Modify jacobian calculation for global strain scalar kernel

### PR DESCRIPTION
**Pull request by SudiptaBiswas**
Use material property  `jacobian_mult` instread of `elasticity_tensor` for jacobian calculation. This approach would be beneficial for multi-phase problems having multiple elasticity tensors. The jacobian multiplier implemented in the stress calculator is used here. 

Refs #11314

### CODE REVIEW

**Review by jwpeterson — APPROVED** [review]
@SudiptaBiswas would you mind giving @mangerij a PR which allows ferret (https://bitbucket.org/mesoscience/ferret) to keep working?

**Comment by dschwen** [comment]
Done in https://bitbucket.org/mesoscience/ferret/commits/8a6713aa7bb29d4923ad8eff6aab9ff9aeba7fc2

**Comment by mangerij** [comment]
CIVET won't merge the current Ferret PR because her changes don't exist yet
:

_dstress_dstrain[_qp] out of scope

Ferret's devel is checked against the MOOSE master before merging with
Ferret's master.

Sudipta should just resend another PR to Ferret after this is merged.

**Comment by mangerij** [comment]
Hmm.

Ferret still seems broken. This time a different error:

https://civet.inl.gov/job/203816/

**Comment by SudiptaBiswas** [comment]
https://civet.inl.gov/job/203816/ is using the older version of ferret. Not with the changes I made in 
https://bitbucket.org/mesoscience/ferret/commits/8a6713aa7bb29d4923ad8eff6aab9ff9aeba7fc2

**Comment by mangerij** [comment]
OK... I see what's going on here.

Just merge this and I will delete that UO in a subsequent PR.

**Comment by SudiptaBiswas** [comment]
This is already merged to devel, will be available in master soon.

**Pull request merged**
